tests/end_to_end/features/steps/multifeature_registration.py

```diff
@@ -0,0 +1,86 @@
+import os
+import random
+
+import requests
+from behave import *
+import featureform as ff
+
+
+@when("I register postgres")
+def step_impl(context):
+    try:
+        context.postgres = context.featureform.register_postgres(
+            name="postgres-quickstart",
+            host="host.docker.internal",  # The docker dns name for postgres
+            port="5432",
+            user=context.POSTGRES_USER,
+            password=context.POSTGRES_PASSWORD,
+            database="postgres",
+        )
+    except Exception as e:
+        context.exception = e
+
+
+@when("I register a table from postgres")
+def step_impl(context):
+    context.transactions = context.postgres.register_table(
+        name="transactions",
+        variant=f"variant_multifeature",
+        table="transactions",  # This is the table's name in Postgres
+    )
+
+
+@when("I create a dataframe from a serving client")
+def step_impl(context):
+    context.client = context.featureform.Client(host="localhost:7878", insecure=True)
```

Comment:
This is going to fail

Reply:
You should reuse statements that already exist. Theres one that already does this in the client.py step file